easybuild/easyconfigs/p/pFUnit/pFUnit-4.2.0-iimpi-2021a.eb

```
@@ -0,0 +1,65 @@
+easyblock = 'CMakeMake'
+
+name = 'pFUnit'
+version = '4.2.0'
+local_fargparse_major_minor_version = '1.1'
+local_gftl_major_minor_version = '1.3'
+local_gftl_shared_major_minor_version = '1.2'
+
+homepage = 'https://github.com/Goddard-Fortran-Ecosystem/pFUnit'
+description = """pFUnit is a unit testing framework enabling JUnit-like testing of serial
+ and MPI-parallel software written in Fortran."""
+
+toolchain = {'name': 'iimpi', 'version': '2021a'}
+
+source_urls = ['https://github.com/Goddard-Fortran-Ecosystem/pFUnit/releases/download/v%(version)s']
+sources = [SOURCE_TAR]
+checksums = ['33df62f80cf03827455508b67d53f820ddffa2ec0f1ba999790ff1f87592ce16']
+
+builddependencies = [
+    ('CMake', '3.20.1'),
+]
```

the build on `jsc-zen2` failed due to missing `M4`:

```
-- Using submodule /tmp/boegelbot/pFUnit/4.2.0/iimpi-2021a/pFUnit-4.2.0/extern/fArgParse/extern/gFTL-shared/extern/gFTL
CMake Error at extern/fArgParse/extern/gFTL-shared/extern/gFTL/CMakeLists.txt:19 (message):
  m4 program not found
```

```suggestion
builddependencies = [
    ('CMake', '3.20.1'),
    ('M4', '1.4.18'),
]
```